mne_connectivity/spectral/time.py

```diff
@@ -431,23 +433,31 @@ def _spectral_connectivity(data, method, kernel, foi_idx,
             window_length = np.arange(0., n_c / float(f), 1.0 / sfreq).shape[0]
             half_nbw = mt_bandwidth / 2.
             n_tapers = int(np.floor(mt_bandwidth - 1))
-            _, eigvals = dpss_windows(window_length, half_nbw, n_tapers,
-                                      sym=False)
+            _, eigvals = dpss_windows(window_length, half_nbw, n_tapers)
```

Comment:
Why do we remove `sym=False`?

Reply:
Good point. I checked and it seems that `sym` was introduced in MNE-Python 1.3. I agree that it should be set to `False` to be more correct. Probably erroneously removed it due to some environment having MNE-Python < 1.3 and it somehow made it here.